compose/xui-frontend.yml

```diff
@@ -0,0 +1,63 @@
+---
+version: '2.4'
+
+services:
+  xui-manage-cases:
+    image: "${XUI_MANAGE_CASES_USE_LOCAL-hmctspublic.azurecr.io/}xui/webapp:${XUI_MANAGE_CASES_TAG:-latest}"
+    container_name: xui-manage-cases
+    environment:
+      FEATURE_APP_INSIGHTS_ENABLED: "true"
+      FEATURE_SECURE_COOKIE_ENABLED: "false"
+      FEATURE_PROXY_ENABLED: "false"
+      FEATURE_TERMS_AND_CONDITIONS_ENABLED: "false"
+      FEATURE_HELMET_ENABLED: "true"
+      FEATURE_OIDC_ENABLED: "false"
+      FEATURE_REDIS_ENABLED: "false"
+
+      MICROSERVICE: xui_webapp
+      PROTOCOL: http
+      XUI_ENV: local
+      NODE_CONFIG_ENV: development
+
+      SERVICES_CCD_COMPONENT_API: http://localhost:3453
+      SERVICES_CCD_DATA_STORE_API: http://ccd-data-store-api:4452
+      SERVICES_IDAM_API_URL: "${IDAM_STUB_SERVICE_NAME:-http://idam-api:5000}"
+      SERVICES_IDAM_CLIENT_ID: xuiwebapp
+      SERVICES_IDAM_LOGIN_URL: "${IDAM_STUB_LOCALHOST:-http://localhost:3501}"
+      SERVICES_IDAM_INDEX_URL: /
+      SERVICES_IDAM_OAUTH_CALLBACK_URL: /oauth2/callback
+      SERVICES_S2S: http://service-auth-provider-api:8080
+      REDISCLOUD_URL: http://localhost:6780
+
+      HEALTH_CCD_COMPONENT_API: http://localhost:3453/health
+      HEALTH_CCD_DATA_API: http://ccd-data-store-api:4452/health
+
+      APPINSIGHTS_INSTRUMENTATIONKEY: TESTVAR
+      IDAM_SECRET: xuiwebapp_secrect
+      S2S_SECRET: "${BEFTA_S2S_CLIENT_SECRET_OF_XUI_WEBAPP}"
+
+      LAUNCH_DARKLY_CLIENT_ID: 1
+      NOW: "true"
+    #      DEBUG: xuiNode:*
+    #      HEALTH_DOCUMENTS_API:
+    #      HEALTH_EM_ANNO_API:
+    #      HEALTH_TERMS_AND_CONDITIONS_API:
+    #      SERVICES_IDAM_ISS_URL:
+    #      SERVICES_MARKUP_API:
+    #      SERVICES_PAYMENTS_URL:
+    #      SERVICES_TERMS_AND_CONDITIONS:
+    #      SERVICES_DOCUMENTS_API: https://api-gateway.dev.dm.reform.hmcts.net
+    #      SERVICES_EM_ANNO_API: https://em-anno-aat.service.core-compute-aat.internal
+    ports:
+      - 3455:3000
+    volumes:
+      - /tmp/sessions:/tmp/sessions
```

Comment:
Do we need this volume to a temporary dir? If so better to use a named one rather than an explicit path to something in `/tmp` to ensure it works for everyone?

Reply:
removed this